bob.yaml

```diff
@@ -18,8 +18,8 @@ node:
 
 chain:
   url: ws://127.0.0.1:8545
-  adjudicator: deploy
-  assetHolder: deploy
+  adjudicator: 0x94503E14E26A433C0802E04f2aC1Bb1CE77321F5
```

Comment:
Why did you change these? Same with `alice.yaml`?!

Reply:
https://github.com/perun-network/perun-eth-demo/pull/17#issue-450349810

Reply:
I think we should not change the addresses, better just change the `README` to start bob first. `dst-go` already uses the demo to deploy the contracts and uses hard-coded addresses :laughing:  
I also used it in the `prnm` tests so keeping the contract addresses would be good.

Reply:
i think you should definitely not rely on these contract addresses to be fixed. so these are todo's for the other projects to remove these dependencies. in fact, i think there should be no such dependencies on a demo project after all. the demo must be independent and it must be possible to change the demo at all times, e.g., for demonstrating additional use cases, or for any other purpose you can think of.

Reply:
`alice.yaml` and `bob.yaml` are just example configuration files that are used in the walkthrough example in the `README`. I think it doesn't matter whether it is changed in the yaml files or README, but it needs to be consistent so that anyone can follow the walkthrough and ends up with two working CLI demo clients.

What do you mean @matthiasgeihs by "i think you should definitely not rely on these contract addresses to be fixed."?

Reply:
After a short clarifying discussion with @matthiasgeihs I agree that it is more intuitive to first start Alice, then Bob and that other projects cannot rely on the yaml files not changing. So I also vote for changing the yamls and keep the more intuitive order in the `README` file.

Reply:
Then we jest rename Alice->Bob and Bob->Alice :raised_hands:

Reply:
Sure, but it is more important that the README file is correct and that it is more intuitive to use (first Alice, then Bob).

Reply:
see latest commit for a resolution

- to keep starting order and contract addresses:
  - swap wallet secret keys in yaml files
  - swap addresses in yaml files